tests/integration.spec.ts

```diff
@@ -163,6 +163,67 @@ describe('jmap-client-ts', () => {
     expect(response).toMatchObject(expect.objectContaining(expected));
   });
 
+  it('should submit email', async () => {
+    const getMailboxesResponse = await client.mailbox_get({
+      accountId: client.getAccountIds()[0],
+      ids: null,
+    });
+
+    const draftMailbox = <string>(
+      getMailboxesResponse.list.find(mailbox => mailbox.name.toLowerCase() === 'drafts')?.id
+    );
```

Comment:
Quick tip: James supports `Mailbox/query` for role mailboxdiscovery.

```
{
  "using": [
    "urn:ietf:params:jmap:core",
    "urn:ietf:params:jmap:mail"],
  "methodCalls": [[
    "Mailbox/query",
    {
      "accountId": "unknownAccountId",
      "filter": {"role":"Draft"}
    },
    "c1"]]
}
```

Would have this done in 1 request.

Reply:
Good point, but there is no such functionality at the moment. And adding functionality for test to be better is out of this PR scope. IMO.

Reply:
Yes yes I know, but I felt compeled to mention it :-)